core/objects.go

```diff
@@ -56,6 +59,18 @@ const (
 	IdentifierDNS = IdentifierType("dns")
 )
 
+// The types of ACME resources
+const (
+	ResourceNewReg        = AcmeResource("new-reg")
+	ResourceNewAuthz      = AcmeResource("new-authz")
+	ResourceNewCert       = AcmeResource("new-cert")
+	ResourceRevokeCert    = AcmeResource("revoke-cert")
+	ResourceRegistration  = AcmeResource("reg")
+	ResourceAuthorization = AcmeResource("authz")
```

Comment:
We would need these if we ever support POST to a reg/authz resource.  I would prefer to keep them, since they're harmless.